codespell_lib/data/dictionary.txt

```diff
@@ -34083,6 +34098,9 @@ unrestrcited->unrestricted
 unrgesiter->unregister
 unroated->unrotated
 unrosponsive->unresponsive
+unrotaete->unrotate
+unrotaeted->unrotated
+unrotaetes->unrotates
```

Comment:
I suggest you drop these. Suggested words are misspelling in major & reputable online dictionaries.
They are misspelling in aspell, hunspell, ispell, chrome browser, aoo-mozilla-en-dict, etc.

https://www.google.com/search?q=define+unrotate
https://www.lexico.com/search?filter=en_dictionary&query=unrotate
https://www.ldoceonline.com/spellcheck/english/?q=unrotate
https://www.oxfordlearnersdictionaries.com/spellcheck/english/?q=unrotate

https://dictionary.cambridge.org/spellcheck/english/?q=unrotate
https://www.dictionary.com/misspelling?term=unrotate
https://www.merriam-webster.com/dictionary/unrotate

https://www.lexico.com/search?filter=en_dictionary&query=unrotated
https://www.lexico.com/search?filter=en_dictionary&query=unrotates

Reply:
[9ff34f6](https://github.com/codespell-project/codespell/pull/2430/commits/9ff34f68a714dd26407bd56c9fcbeddd96e911da)